bridges/FilterBridge.php

```diff
@@ -2,80 +2,115 @@
 
 class FilterBridge extends FeedExpander {
 
-	const MAINTAINER = 'Frenzie';
+	const MAINTAINER = 'Frenzie, ORelio';
 	const NAME = 'Filter';
 	const CACHE_TIMEOUT = 3600; // 1h
 	const DESCRIPTION = 'Filters a feed of your choice';
-	const URI = 'https://github.com/rss-bridge/rss-bridge';
+	const URI = 'https://github.com/RSS-Bridge/rss-bridge';
 
 	const PARAMETERS = array(array(
 		'url' => array(
 			'name' => 'Feed URL',
 			'required' => true,
 		),
 		'filter' => array(
-			'name' => 'Filter item title (regular expression)',
+			'name' => 'Filter (regular expression)',
+			'required' => false,
+		),
+		'case_insensitive' => array(
+			'name' => 'Case-insensitive regular expression',
+			'type' => 'checkbox',
 			'required' => false,
 		),
 		'filter_type' => array(
 			'name' => 'Filter type',
 			'type' => 'list',
 			'required' => false,
 			'values' => array(
-				'Permit' => 'permit',
-				'Block' => 'block',
+				'Keep matching items' => 'permit',
+				'Hide matching items' => 'block',
 			),
 			'defaultValue' => 'permit',
 		),
+		'filter_target' => array(
+			'name' => 'Apply regex on field',
+			'type' => 'list',
+			'required' => false,
+			'values' => array(
+				'Title' => 'title',
+				'Content' => 'content',
+				'Title and Content' => 'title_content',
+			),
+			'defaultValue' => 'title',
+		),
```

Comment:
I have case where author also should be filtered. If we add it `filter_target` values it will become messy.
Instead of this, I would suggest to make 3 checkboxes like `Apply filter to title` (default yes), `Apply filter to body` (default no), `Apply filter to author` (default no). In this case you also need to add check in collectData method

Reply:
Good idea, doing that. However no need to change `collectData` because `parseItem` can hide an item by returning `null`.